lib/src/widgets/button.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_platform_widgets/flutter_platform_widgets.dart';
+import 'package:oneplace_illinois/src/misc/colors.dart';
+
+class Button extends StatelessWidget {
+  final Widget child;
+  final void Function()? onPressed;
+
+  Button({required this.child, this.onPressed});
+
+  @override
+  Widget build(BuildContext context) {
+    return PlatformButton(
+      onPressed: onPressed,
+      child: child,
+      color: Colors.white,
```

Comment:
Probably make this a property like the `child` property, just in case. Set the default to white.